pkg/agent/config.go

```diff
@@ -57,6 +68,354 @@ type VenafiCloudConfig struct {
 	UploadPath string `yaml:"upload_path,omitempty"`
 }
 
+type AgentCmdFlags struct {
```

Comment:
**Self-review:** these were defined as package-level variables, so I moved them to be a structs so that I can test `getConfiguration`.